node/actors/network/src/lib.rs

```diff
@@ -152,6 +162,8 @@ impl Runner {
                 }
             }
 
+            // TODO: check if we are active attester to get new L1 Batches, sign them and broadcast the signature
```

Comment:
What happens with L1 batches that aren't new but no submitter has managed either to gather enough signatures over the gossip channel or to submit it to L1. Would attesters eventually re-broadcast their signatures?

Reply:
Thanks; IIUC the broadcasts and gossips continue until the cert appears on L1.

This is what I wanted to confirm. I asked @brunoffranca after watching one of his presentations; suggested looking at this PR.